src/main.rs

```diff
@@ -83,20 +83,25 @@ async fn main() {
 
     match &cli.command {
         Commands::Translate(args) => {
-            let source_path = args.path.as_ref().unwrap();
-            if !(Path::new(&source_path).exists()) {
+            let source_filepath_str = args.path.as_ref().unwrap();
+            let source_filepath = Path::new(&source_filepath_str);
+
+            if !source_filepath.exists() {
                 println!("Provided filepath does not exist");
                 return;
-            }
+            };
 
             let target_language = args.target_language;
             let source_language = args.source_language;
 
-            let target_file_path = format!("strings/strings_{:#?}.json", target_language);
+            let parent_dir = source_filepath.parent().unwrap();
+
+            let target_filename = format!("strings_{:#?}.json", target_language);
+            let target_filepath = parent_dir.join("/").join(target_filename);
 
             let api_key = &args.api_key;
 
-            let source_json = match file::from_json(source_path.to_string()) {
+            let source_json = match file::from_json(source_filepath_str.to_string()) {
```

Comment:
Does `source_filepath_str` need to be used here or should it work with a Rust Path as well?

Reply:
It's a good question. My gut wants to use a Path object here but I can't find anything that says this function will be okay with that. Let's stick to a string for now can clean up later.

Reply:
I just realized that this is a function defined in our module and not a library. Holding off on refactoring that right now.

Reply:
That would explain why I couldn't google it